studio/app/common/dataclass/polar.py

```diff
@@ -33,14 +32,23 @@ def output_path(self) -> OutputPath:
 
     def save_plot(self, output_dir):
         for i in range(len(self.data)):
-            fig = px.line_polar(
-                r=self.data[i],
-                theta=self.columns,
-                direction="counterclockwise",
-                start_angle=0,
-                line_close=True,
+            theta = linspace(0, 2 * pi, len(self.columns))  # Convert theta to radians
+            plt.figure()
+            ax = plt.subplot(111, polar=True)
+            ax.plot(theta, self.data[i], color="#636EFA")
+            ax.plot(
+                [theta[-1], theta[0]],
+                [self.data[i][-1], self.data[i][0]],
+                color="#FFA15A",
+                linestyle="-",
+                linewidth=2,
             )
-            plot_file = join_filepath([output_dir, f"{self.file_name}_{i}.png"])
-            pio.write_image(fig, plot_file)
-
-            save_thumbnail(plot_file)
+            ax.set_theta_direction(1)  # Clockwise
+            ax.tick_params(axis="x", rotation="auto")
+            ax.set_facecolor("white")
+            fig = plt.gcf()
+            fig.set_facecolor("white")
+            plt.tight_layout()
+            plot_file = join_filepath([output_dir, f"{self.filename}{i}.png"])
```

Comment:
here is an exception at runtime.
please check and correct.
*check other areas as well.